### PR TITLE
[ALLUXIO-2307]Update LeaderSelectorClient.java

### PR DESCRIPTION
**Pull request by yekang**
https://alluxio.atlassian.net/browse/ALLUXIO-2307

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: core/common/src/main/java/alluxio/LeaderSelectorClient.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public LeaderSelectorClient(String zookeeperAddress, String electionPath, String
     mLeaderSelector = new LeaderSelector(getNewCuratorClient(), mElectionPath, this);
     mLeaderSelector.setId(name);
 
-    // for most cases you will want your instance to requeue when it relinquishes leadership
+    // For most cases you will want your instance to requeue when it relinquishes leadership
```
Comment (aaudiber) [inline core/common/src/main/java/alluxio/LeaderSelectorClient.java]: This also needs a period at the end.

Comment (yekang) [inline core/common/src/main/java/alluxio/LeaderSelectorClient.java]: thank you

**Pull request merged**